# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: third_party/rust/naga/src/front/spv/error.rs
# Commit: b3109ad14685
# Full Hash: b3109ad14685624ada7861083ff5332aec9a7818
# Author: Andy Leiserson <aleiserson@mozilla.com>
# Date: 2025-10-07 17:00 +0000
# Description:
#   Bug 1992842 - Update wgpu to upstream 1072b878 (2025-10-06) r=webgpu-reviewers,supply-chain-reviewers,ErichDonGubler
# ==============================================================================

diff -r 1d3f000cf1f9 -r b3109ad14685 third_party/rust/naga/src/front/spv/error.rs
--- a/third_party/rust/naga/src/front/spv/error.rs	Tue Oct 07 16:51:44 2025 +0000
+++ b/third_party/rust/naga/src/front/spv/error.rs	Tue Oct 07 17:00:40 2025 +0000
@@ -107,6 +107,8 @@
     InvalidGlobalVar(crate::Expression),
     #[error("invalid image/sampler expression {0:?}")]
     InvalidImageExpression(crate::Expression),
+    #[error("image write without format is not currently supported. See https://github.com/gfx-rs/wgpu/issues/6797")]
+    InvalidImageWriteType,
     #[error("invalid image base type {0:?}")]
     InvalidImageBaseType(Handle<crate::Type>),
     #[error("invalid image {0:?}")]